__tests__/api.test.js

```diff
@@ -353,3 +353,32 @@ describe("GET /api/articles (topic query)", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comment_count", () => {
+  test("GET:200 responds with count of all comments for specific article", () => {
+    return request(app)
+      .get("/api/articles/1/comment_count")
```

Comment:
The making of this endpoint was not the task. 

- The task was to add a comment_count column for when fetching individual articles by their article id - like you do when you are fetching all articles. Please make sure you are reading the instructions properly.